deploy/backplane/acs/03-acs-rhacs-observability-role.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backplane-acs-rhacs-observability
+  namespace: openshift-monitoring
+rules:
+# ACS SRE can vew monitoring resources
+- apiGroups:
+  - monitoring.coreos.com
+  resources:
+  - alertmanagerconfigs
+  - alertmanagers
+  - podmonitors
+  - probes
+  - prometheuses
+  - prometheusrules
+  - servicemonitors
+  - thanosrulers
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can view observabilities
+- apiGroups:
+  - observability.redhat.com
+  resources:
+  - observabilities
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can view grafanas
+- apiGroups:
+  - integreatly.org
+  resources:
+  - grafanas
+  - grafanadatasources
+  - grafanadashboards
+  verbs:
+  - get
+  - list
+  - watch
+# ACS SRE can patch persistentvolumeclaims
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - patch
+# ACS SRE can restart statefulsets
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - patch
```

Comment:
No get/list/watch?

Reply:
Added it to the backplane-acs-admins-cluster-aggregate role